sqlite_migration.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
1. Please rename ``sqlitedict_migration``, this is not unique enough to be discoverable.
2. Consider distributing in setup.py so that it may be called by a user who has installed sqlitedict from pypi, and not using a source code repository like you or I. I think it is unreasonable to expect users to clone a github repository to migrate a database.